config/locales/en.yml

```diff
@@ -108,10 +108,13 @@ en:
         resend_unlock_instructions_header: "Resend unlock instructions"
         resend_unlock_instructions_button: "Resend unlock instructions"
     passwords:
+      password:
```

Comment:
please, check `ActiveRecord` translations https://guides.rubyonrails.org/i18n.html#looking-up-translations

Reply:
used thanslations in tests